Library/Homebrew/os/mac/mach.rb

```diff
@@ -202,3 +199,7 @@ def dylib_id
     mach_metadata.dylib_id
   end
 end
+
+class Pathname
+  include(MachO)
+end
```

Comment:
How about `Library/Homebrew/os/mac/extend/` or something?

If we still like to put it in original place, may be as `include(MachO) if OS.mac?`?

If you don't like either of these, I'm happy to keep original form. (Just make sure on the same page for style issue)